hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/SnapshotDeletingService.java

```diff
@@ -89,16 +86,17 @@ public class SnapshotDeletingService extends AbstractKeyDeletingService {
 
   private final OzoneManager ozoneManager;
   private final OmSnapshotManager omSnapshotManager;
-  private final SnapshotChainManager chainManager;
+  private final SnapshotChainManager snapshotChainManager;
   private final AtomicBoolean suspended;
   private final OzoneConfiguration conf;
   private final AtomicLong successRunCount;
-  private final long snapshotDeletionPerTask;
-  private final int keyLimitPerSnapshot;
+  private final int keyLimitPerTask;
+  private final int snapshotDeletionPerTask;
```

Comment:
scope change from long to int. Similar comment as above regarding renaming variable.

Reply:
we need to rename these variables, since the definition of these variables have changed.